pkg/api/operatorclient.go

```diff
@@ -1,4 +1,4 @@
-package operator2
+package api
```

Comment:
why not pkg/operator2/client ?

Reply:
Traditionally api packages contain apis.

Reply:
You are right! moved it under `pkg/operator2/client`